### DISCUSSION TITLE
請問是否能讓字幕翻譯功能新增可以批次處理

### DISCUSSION THREAD

**Discussion by Jackchen1005** [immersive-translate/immersive-translate#2084] [Ideas]
### 新功能 | New Feature

1.希望字幕可以一次上傳比如200個Srt, 然後全部批次處理
2.
3.

### 为什么您想要添加这些功能 | Why you want to add these features

1.因為英文字幕太多，希望可以批次處理，謝謝
2.
3.

**Comment by theowenyoung** [comment]
#1422 见次issue，未来应该会支持。
